tests/integration/modules/test_file.py

```diff
@@ -179,9 +179,14 @@ def test_remove_broken_symlink(self):
 
     def test_cannot_remove(self):
         ret = self.run_function('file.remove', arg=['tty'])
-        self.assertEqual(
-            'ERROR executing \'file.remove\': File path must be absolute: tty', ret
-        )
+        if salt.utils.platform.is_windows():
```

Comment:
Should we just normalize this error message in the module itself instead?

Reply:
Assuming we require an absolute path for both platforms. I think we do but I'll verify that and then change the message if needed.